Shared/mods/deathmatch/logic/luadefs/CLuaMatrixDefs.cpp

```diff
@@ -91,7 +95,7 @@ int CLuaMatrixDefs::Create(lua_State* luaVM)
             argStream.ReadVector3D(vecRotation);
             ConvertDegreesToRadiansNoWrap(vecRotation);
             matrix = CMatrix(vecPosition, vecRotation);
-        }
+        }        
```

Comment:
I think you've introduced some unwanted whitespace here 🙂

Reply:
Oh, my bad